src/components/accordion/accordion.d.ts

```diff
@@ -28,6 +28,10 @@ export interface AccordionProps extends SpaceProps {
   iconType?: "chevron_down" | "dropdown";
   /** Sets icon alignment - accepted values: 'left', 'right' (default) */
   iconAlign?: AlignBinaryType;
+  /** Sets accordion type to either primary (default), or secondary */
+  type?: ThemesBinary;
```

Comment:
Did you mean to add these or is it from the rebase?

Reply:
Thats from a rebase, I did not add that. I can reverse it out though